client/src/components/CustomerTable.jsx

```diff
@@ -15,8 +16,12 @@ const CustomerTable = () => {
   const fetchCustomers = async () => {
     const fetch = await axios.get(`http://localhost:1337/api/customers/`)
 
-    console.log(fetch)
-    setCustomers(fetch.data)
+    if (fetch.status !== 200) {
+      // Set a Notifcation state here
+    } else {
+      console.log(fetch)
+      setCustomers(fetch.data)
+    }
   }
 
   const updatedSelectedCustomers = (selection) => {
```

Comment:
Every function that is called within the JSX part should use a ´useCallback´or ´useMemo´ to avoid having to redeclare the function on every render.